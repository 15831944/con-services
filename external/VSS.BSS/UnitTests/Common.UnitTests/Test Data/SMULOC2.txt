<?xml version="1.0" encoding="UTF-8"?>
<smuLoc:topic xmlns:smuLoc="http://www.cat.com/em/feed/v2/ServiceMeterLocation" xmlns:feed="http://www.cat.com/em/feed/v2/Common" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cat.com/em/feed/v2/ServiceMeterLocation https://emfeed.cat.com/EMFeedFiles/schema/v2/ServiceMeterLocation.xsd">
<nextBuffer>
<url></url>
<moreData>false</moreData>
</nextBuffer>
  <!-- Valid -->
   <message>
      <header>
         <messageId>0</messageId>
         <masterMsgId>119315420</masterMsgId>
         <equipment>
            <serialNumber>-MWP02718</serialNumber>
            <make>CAT</make>
         </equipment>
         <moduleCode>-DQ0200079955DQ1</moduleCode>
         <moduleTime>2009-09-16 23:49:14</moduleTime>
         <receivedTime>2009-09-17 05:00:13</receivedTime>
      </header>
      <serviceMeter>
         <value>1563</value>
         <uom>Hours</uom>
      </serviceMeter>
      <location>
         <latitude>44.05657</latitude>
         <longitude>-75.7777</longitude>
         <validity>V</validity>
      </location>
   </message>
   <!-- Invalid - validity specified as invalid -->
   <message>
      <header>
         <messageId>0</messageId>
         <masterMsgId>119315434</masterMsgId>
         <equipment>
            <serialNumber>-N1A01456</serialNumber>
            <make>CAT</make>
         </equipment>
         <moduleCode>-DQ0200020816Q1</moduleCode>
         <moduleTime>2009-09-16 23:26:59</moduleTime>
         <receivedTime>2009-09-17 05:00:21</receivedTime>
      </header>
      <serviceMeter>
         <value>3927</value>
         <uom>Hours</uom>
      </serviceMeter>
      <location>
         <latitude>50.32887</latitude>
         <longitude>7.429033</longitude>
         <validity>I</validity>
      </location>
   </message>
   <!-- Valid - missing validityspecified block - assumed to be good -->
   <message>
      <header>
         <messageId>0</messageId>
         <masterMsgId>119315421</masterMsgId>
         <equipment>
            <serialNumber>-MWP02718</serialNumber>
            <make>CAT</make>
         </equipment>
         <moduleCode>-DQ0200079955DQ1</moduleCode>
         <moduleTime>2009-09-16 23:50:14</moduleTime>
         <receivedTime>2009-09-17 05:01:13</receivedTime>
      </header>
      <serviceMeter>
         <value>1563</value>
         <uom>Hours</uom>false
      </serviceMeter>
      <location>
         <latitude>44.05657</latitude>
         <longitude>-75.81073</longitude>         
      </location>
   </message>
   <!-- Invalid - missing location block -->
   <message>
      <header>
         <messageId>0</messageId>
         <masterMsgId>119315421</masterMsgId>
         <equipment>
            <serialNumber>-MWP02718</serialNumber>
            <make>CAT</make>
         </equipment>
         <moduleCode>-DQ0200079955DQ1</moduleCode>
         <moduleTime>2009-09-16 23:51:14</moduleTime>
         <receivedTime>2009-09-17 05:02:13</receivedTime>
      </header>
      <serviceMeter>
         <value>1563</value>
         <uom>Hours</uom>
      </serviceMeter>      
   </message>
   <message>
      <header>
         <messageId>0</messageId>
         <masterMsgId>119315421</masterMsgId>
         <equipment>
            <serialNumber>-MWP02718</serialNumber>
            <make>CAT</make>
         </equipment>
         <moduleCode>-DQ0200079955DQ1</moduleCode>
         <moduleTime>2001-09-16 23:50:14</moduleTime>
         <receivedTime>2009-09-17 05:01:13</receivedTime>
      </header>
      <serviceMeter>
         <value>1563</value>
         <uom>Hours</uom>
      </serviceMeter>
      <location>
         <latitude>44.05657</latitude>
         <longitude>-75.81073</longitude>         
      </location>
   </message>
 </smuLoc:topic>
