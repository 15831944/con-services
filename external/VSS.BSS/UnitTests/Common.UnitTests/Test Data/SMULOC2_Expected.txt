<?xml version="1.0" encoding="utf-8"?><ArrayOfNHDataWrapper xmlns:i="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.nighthawk.com/nighthawk/service/NHData/2009/10"><NHDataWrapper><Data z:Id="i1" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataHoursLocation" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:Altitude i:nil="true" /><d3p1:AssetID>1290</d3p1:AssetID><d3p1:CustomerID>0</d3p1:CustomerID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:EventUTC>2009-09-16T23:49:14</d3p1:EventUTC><d3p1:GPSDeviceID>-DQ0200079955DQ1</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:Latitude>44.05657</d3p1:Latitude><d3p1:LocIsValid>true</d3p1:LocIsValid><d3p1:LocationAge i:nil="true" /><d3p1:LocationUncertaintyMeters i:nil="true" /><d3p1:Longitude>-75.7777</d3p1:Longitude><d3p1:OdometerMiles i:nil="true" /><d3p1:RuntimeHours>1563</d3p1:RuntimeHours><d3p1:SourceMsgID>119315420</d3p1:SourceMsgID><d3p1:SpeedMPH i:nil="true" /><d3p1:Track i:nil="true" /><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i2" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataHoursLocation" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:Altitude i:nil="true" /><d3p1:AssetID>1291</d3p1:AssetID><d3p1:CustomerID>0</d3p1:CustomerID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:EventUTC>2009-09-16T23:26:59</d3p1:EventUTC><d3p1:GPSDeviceID>-DQ0200020816Q1</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:Latitude>50.32887</d3p1:Latitude><d3p1:LocIsValid>false</d3p1:LocIsValid><d3p1:LocationAge i:nil="true" /><d3p1:LocationUncertaintyMeters i:nil="true" /><d3p1:Longitude>7.429033</d3p1:Longitude><d3p1:OdometerMiles i:nil="true" /><d3p1:RuntimeHours>3927</d3p1:RuntimeHours><d3p1:SourceMsgID>119315434</d3p1:SourceMsgID><d3p1:SpeedMPH i:nil="true" /><d3p1:Track i:nil="true" /><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i3" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataHoursLocation" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:Altitude i:nil="true" /><d3p1:AssetID>1290</d3p1:AssetID><d3p1:CustomerID>0</d3p1:CustomerID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:EventUTC>2009-09-16T23:50:14</d3p1:EventUTC><d3p1:GPSDeviceID>-DQ0200079955DQ1</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:Latitude>44.05657</d3p1:Latitude><d3p1:LocIsValid>true</d3p1:LocIsValid><d3p1:LocationAge i:nil="true" /><d3p1:LocationUncertaintyMeters i:nil="true" /><d3p1:Longitude>-75.81073</d3p1:Longitude><d3p1:OdometerMiles i:nil="true" /><d3p1:RuntimeHours>1563</d3p1:RuntimeHours><d3p1:SourceMsgID>119315421</d3p1:SourceMsgID><d3p1:SpeedMPH i:nil="true" /><d3p1:Track i:nil="true" /><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i4" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataHoursLocation" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:Altitude i:nil="true" /><d3p1:AssetID>1290</d3p1:AssetID><d3p1:CustomerID>0</d3p1:CustomerID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:EventUTC>2009-09-16T23:51:14</d3p1:EventUTC><d3p1:GPSDeviceID>-DQ0200079955DQ1</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:Latitude i:nil="true" /><d3p1:LocIsValid>false</d3p1:LocIsValid><d3p1:LocationAge i:nil="true" /><d3p1:LocationUncertaintyMeters i:nil="true" /><d3p1:Longitude i:nil="true" /><d3p1:OdometerMiles i:nil="true" /><d3p1:RuntimeHours>1563</d3p1:RuntimeHours><d3p1:SourceMsgID>119315421</d3p1:SourceMsgID><d3p1:SpeedMPH i:nil="true" /><d3p1:Track i:nil="true" /><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper></ArrayOfNHDataWrapper>